operators/advanced_network/managers/dpdk/adv_network_dpdk_mgr.cpp

```diff
@@ -417,17 +436,23 @@ void DpdkMgr::initialize() {
   int max_rx_batch_size = 0;
```

Comment:
Don't map memory regions if running in SW loopback mode, `rte_eth_dev` calls will fail.
```suggestion
  if (!loopback_) {
    if (map_mrs() != AdvNetStatus::SUCCESS) {
      HOLOSCAN_LOG_CRITICAL("Failed to map MRs");
      return;
    }
  }
```

Reply:
Weird I didn't see a problem here. Was it showing up for you?

Reply:
Yes, on my setup with no NIC, I get the following error:
```
Invalid port_id=0
[2024-12-10 20:32:46.176] [623] [holoscan] [critical] [adv_network_dpdk_mgr.cpp:164] Failed to get device info for port 0
```